docker-assets/README.md

```diff
@@ -11,6 +11,13 @@ It needs to be initiated from the root directory of the manageiq git repository
 docker build -t manageiq .
 ```
 
+To build versions other than the tip of master:
+
+```
+docker build -t manageiq:darga --build-arg REF=darga .                 # From tip of darga branch
+docker build -t manageiq:darga-1-beta1 --build-arg REF=darga-1-beta1 . # From darga-1-beta1 tag
+```
+
```

Comment:
For the sake of doc completeness perhaps adding an example in how to build your own branch/clone uncomment "#ADD ." and commenting #curl layer.